gap_darwin.go

```diff
@@ -0,0 +1,135 @@
+package bluetooth
+
+import (
+	"errors"
+	"time"
+
+	"github.com/JuulLabs-OSS/cbgo"
+)
+
+// Address contains a Bluetooth address, which is a MAC address plus some extra
+// information.
+type Address struct {
+	// UUID if this is macOS.
+	UUID
+
+	isRandom bool
```

Comment:
I assume the `isRandom` bit is included in the UUID and isn't exposed.

Reply:
Since never "random" per se, changed to show that in the implementation.